src/app/client/src/app/modules/announcement/announcement-routing.module.ts

```diff
@@ -7,13 +7,15 @@ import { OutboxComponent, DeleteComponent, DetailsComponent, DetailsPopupCompone
 const routes: Routes = [
   {
     path: 'announcement/outbox/:pageNumber', component: OutboxComponent, canActivate: [AuthGuard],
+    data: [{label: 'Home', url: '/home'}, {label: 'Profile', url: '/profile'}, {label: 'My Announcements', url: ''}],
```

Comment:
data should be in 
data: {breadcrumbs: [{label: 'Home', url: '/home'}, {label: 'Profile', url: '/profile'}, {label: 'My Announcements', url: ''}]} in all places